js/packages/web/src/pages/_app.tsx

```diff
@@ -9,7 +9,7 @@ function App({ Component, pageProps }: AppProps) {
       <Head>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
-      {typeof window === 'undefined' ? null : <Component {...pageProps} />}
```

Comment:
I think they have this because items like phantom aren't accessible unless the render is on the browser. Next does the SSR rendering so all things can be rendered on the server or client.

Reply:
Though for you're goal this is desirable. 

From my working with next I think what is the index.tsx isMounted suffices for the phantom being available.